blog/electron-36-0.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Electron 36.0.0
+date: 2025-04-28T00:00:00.000Z
+authors:
+  - georgexu99
+slug: electron-36-0
+tags: [release]
+---
+
+Electron 36.0.0 has been released! It includes upgrades to Chromium 136, V8 13.6, and Node 22.14.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 36.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Bluesky](https://bsky.app/profile/electronjs.org) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
```

Comment:
Just to make sure: Is anyone actually tracking feedback through Bluesky, Mastodon, or Discord? I've never really heard anyone talk about them much.

Reply:
@nikwen I think it's up to individual maintainer efforts but I usually look at our Bluesky notifications at least once a week!

Reply:
Okay, great! Thanks for the quick reply. I just wanted to make sure we don't mention a channel that doesn't get looked at.